src/components/NumberInput.tsx

```diff
@@ -16,6 +16,7 @@ function NumberInput(props: NumberInputProps) {
         min={props.min}
         max={props.max}
         onChange={(e) => props.onChange(Number(e.target.value))}
+        style={{ caretColor: 'transparent' }}
```

Comment:
could you please use CSS classes to stylize elements? inline styles are bad practice